knimin/tests/test_ag_pulldown.py

```diff
@@ -73,6 +73,11 @@ def test_post(self):
                        ' for file download. It may take a while with many '
                        'barcodes.</h3>'), response.body)
 
+        data = {'external': 'ab', 'external': 'cd'}
```

Comment:
You got me :-) Seems like I don't understand what is going on here. From the handler code `hold = self.get_arguments('external', [])
        if hold:
            external = ','.join(hold)
        else:
            external = ''`
I infer that `external` can be a list instead of a single value. But how can I pass a list via `multipart_post`? My approach does not seem to work.